src/test/java/jenkins/plugins/openstack/pipeline/OpenstackMachineStepTest.java

```diff
@@ -97,11 +98,14 @@ public void bootInvalidTemplate() throws Exception {
     @Test
     public void checkSerializableSimplifiedServer() throws Exception {
 
+        final String echoIpStepWindows = "bat \"echo Instance IP: ${srv.address}\"";
+        final String echoIpStepUnix = "sh \"echo Instance IP: ${srv.address}\"";
+        final String echoIpStep = Functions.isWindows() ? echoIpStepWindows : echoIpStepUnix;
         WorkflowJob bootSerializableCheck = j.jenkins.createProject(WorkflowJob.class, "bootSerializableCheck");
         bootSerializableCheck.setDefinition(new CpsFlowDefinition(
                 " def srv = openstackMachine cloud: 'openstack', template: 'template0'\n" +
                         "node ('master') { \n" +
-                        "  sh \"echo Instance IP: ${srv.address}\" \n" +
```

Comment:
I've yet to learn about pipelines so this syntax is all new to me :-)
Thanks for the URL - it made making the code changes much easier.